flight/targets/sparky/fw/pios_config.h

```diff
@@ -32,33 +32,34 @@
 #include <pios_flight_config.h>
 
 /* Major features */
+#define STABILIZATION_LQG
 
 /* Enable/Disable PiOS Modules */
 #define PIOS_INCLUDE_DMA_CB_SUBSCRIBING_FUNCTION
 
 #define PIOS_INCLUDE_I2C
-#define PIOS_INCLUDE_CAN
 
 /* Select the sensors to include */
-#define PIOS_INCLUDE_MS5611
-#define PIOS_INCLUDE_MPXV5004
-#define PIOS_INCLUDE_MPXV7002
 #define PIOS_INCLUDE_MPU
 #define PIOS_INCLUDE_MPU_MAG
+#define PIOS_INCLUDE_MS5611
 #define PIOS_INCLUDE_HMC5883
 #define PIOS_INCLUDE_HMC5983_I2C
-#define PIOS_TOLERATE_MISSING_SENSORS
 
 /* Com systems to include */
 #define PIOS_INCLUDE_MAVLINK
 #define PIOS_INCLUDE_LIGHTTELEMETRY 
 
-
 /* Supported receiver interfaces */
 #define PIOS_INCLUDE_PWM
 
-
 /* Flags that alter behaviors */
+#define AUTOTUNE_AVERAGING_DECIMATION 2
+
+#undef PIOS_INCLUDE_CROSSFIRE
```

Comment:
Can we keep the rx protocol without the telemetry?

Reply:
I rebuilt with Crossfire enabled.  Sparky still boots, seems to run fine.  I can't test Crossfire though.  Push the updated files.